pkg/cmd/provisioning/aws/create_all.go

```diff
@@ -103,6 +104,7 @@ func NewCreateAllCmd() *cobra.Command {
 	createAllCmd.PersistentFlags().StringVar(&CreateAllOpts.CredRequestDir, "credentials-requests-dir", "", "Directory containing files of CredentialsRequests to create IAM Roles for (can be created by running 'oc adm release extract --credentials-requests --cloud=aws' against an OpenShift release image)")
 	createAllCmd.MarkPersistentFlagRequired("credentials-requests-dir")
 	createAllCmd.PersistentFlags().StringVar(&CreateAllOpts.TargetDir, "output-dir", "", "Directory to place generated files (defaults to current directory)")
+	createAllCmd.PersistentFlags().BoolVar(&CreateAllOpts.EnableTechPreview, "--enable-tech-preview", false, "Opt into processing CredentialsRequests marked as tech-preview")
```

Comment:
@joelddiaz  "--enable-tech-preview" unnecessary --

Reply:
and here is testcases for the PR https://polarion.engineering.redhat.com/polarion/#/project/OSE/workitems?query=trello%3AOCPBUGSM%5C-38879, please help to review, thanks

Reply:
Yes, the `--` is unnecessary. Pushed an update to the PR. Thanks!